composer.json

```diff
@@ -26,7 +26,7 @@
   },
   "autoload-dev": {
     "psr-4": {
-      "WpOop\\WordPress\\Test\\Unit\\": "tests/unit"
+      "WpOop\\WordPress\\Test\\": "tests"
```

Comment:
Well, they are unit tests, and I think it would be nice if unit tests could be found in the same directory in all projects.

Reply:
They are, it just allows to add other dirs there with this namespace.

Reply:
Unit tests differ from e.g. functional or acceptance tests not just by namespace, but by their purpose. That's why they have separate root namespaces. It's that way in all other Dhii/WP-OOP projects.

Reply:
Then again, like for other similar things: I would love it if you raised this question in a discussion, and just do it the same way as everywhere else for now.

Reply:
Nothing changes here about test types, it was simply to allow other dirs besides `Unit` without duplicating this namespace declaration in `composer.json`.

Reply:
I understand the reason. But there is also a reason for doing it the way I have described, like I explained above. Why is it a problem to just declare another root namespace for another kind of tests, until we have a properly argumented discussion and decision about this?

Reply:
:heavy_check_mark: